searx/engines/bing_videos.py

```diff
@@ -82,22 +84,18 @@ def response(resp):
     dom = html.fromstring(resp.text)
 
     for result in dom.xpath('//div[@class="dg_u"]'):
-        try:
```

Comment:
it is more than pylint, but why not.

Note that should be an issue: create a context manage named "no_exception":
```
for result in dom.xpath('//div[@class="dg_u"]'):
    with no_exception():
        ...
```

-->

exception are hidden EXCEPT in debug mode

Reply:
> it is more than pylint, but why not.

more or less, it is pylint ...  [W0702 (bare-except)](http://pylint-messages.wikidot.com/messages:w0702)

> exception are hidden EXCEPT in debug mode

The problem of this try block is to catch all exception classes, what is an antipattern ...

>  create a context manage named "no_exception":

no please not .. it is a really bad practice to catch all issues / issues in the engines are handle by the caller of the engine.

I tested the bing engines and I can't find an exception, so for what is a try/except block needed? ..